multisig/simulations/vip-002/vip-002-ethereum/simulations.ts

```diff
@@ -0,0 +1,569 @@
+        });
+      }
+    });
+
+    describe("Risk parameters", () => {
+      for (const [symbol, params] of Object.entries(riskParameters) as [VTokenSymbol, RiskParameters][]) {
+        describe(`${symbol} risk parameters`, () => {
+          let vToken: Contract;
+          let comptroller: Contract;
+          let underlyingDecimals: number;
+
+          before(async () => {
+            vToken = await ethers.getContractAt(VTOKEN_ABI, vTokens[symbol]);
+            comptroller = await ethers.getContractAt(COMPTROLLER_ABI, vTokenState[symbol].comptroller);
+            const underlyingAddress = vTokenState[symbol].underlying;
+            const underlying = await ethers.getContractAt(ERC20_ABI, underlyingAddress);
+            underlyingDecimals = await underlying.decimals();
+          });
+
+          it(`should set ${symbol} reserve factor to ${params.reserveFactor}`, async () => {
+            expect(await vToken.reserveFactorMantissa()).to.equal(parseUnits(params.reserveFactor, 18));
+          });
+
+          it(`should set ${symbol} collateral factor to ${params.collateralFactor}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.collateralFactorMantissa).to.equal(parseUnits(params.collateralFactor, 18));
+          });
+
+          it(`should set ${symbol} liquidation threshold to ${params.liquidationThreshold}`, async () => {
+            const market = await comptroller.markets(vTokens[symbol]);
+            expect(market.liquidationThresholdMantissa).to.equal(parseUnits(params.liquidationThreshold, 18));
+          });
+
+          it(`should ${symbol} have correct protocol seize share`, async () => {
+            if (symbol === "vUSDC_Stablecoins" || symbol === "vUSDT_Stablecoins" || symbol === "vcrvUSD_Stablecoins") {
+              expect(await vToken.protocolSeizeShareMantissa()).to.equal(parseUnits("0.01", 18));
+            } else {
+              expect(await vToken.protocolSeizeShareMantissa()).to.equal(parseUnits("0.05", 18));
+            }
+          });
+
+          it(`should set ${symbol} supply cap to ${params.supplyCap}`, async () => {
+            expect(await comptroller.supplyCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.supplyCap, underlyingDecimals),
+            );
+          });
+
+          it(`should set ${symbol} borrow cap to ${params.borrowCap}`, async () => {
+            expect(await comptroller.borrowCaps(vTokens[symbol])).to.equal(
+              parseUnits(params.borrowCap, underlyingDecimals),
+            );
+          });
+        });
+      }
+    });
+
+    describe("Pools configuration", () => {
+      const checkComptroller = (
+        comptrollerAddress: string,
+        comptrollerName: string,
+        liquidationIncentive: BigNumber,
+      ) => {
+        describe(`${comptrollerName} Comptroller`, () => {
+          let comptroller: Contract;
```

Comment:
nit: similar comment about combining the tests to avoid scoping variable unnecessarily outside of tests